.github/workflows/test_iree.yml

```diff
@@ -67,12 +67,49 @@ jobs:
           source ${VENV_DIR}/bin/activate
           pytest iree_tests/onnx/node/generated -n auto -rpfE --timeout=30 --retries 2 --retry-delay 5 --durations=10
 
+  linux_x86_64_w7900_gpu_models:
+    strategy:
+      matrix:
+        os: [nodai-amdgpu-w7900-x86-64]
+    name: Linux (x86_64 w7900) Models GPU
+    runs-on: ${{matrix.os}}
+    env:
+      VENV_DIR: ${{ github.workspace }}/.venv
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@v4
+        with:
+          submodules: false
+          lfs: true
+      - name: "Setting up Python"
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      - name: "Setup Python venv"
+        run: python3 -m venv ${VENV_DIR}
+
+      - name: "Installing IREE nightly release Python packages"
+        run: |
+          source ${VENV_DIR}/bin/activate
+          python3 -m pip install \
+            --find-links https://iree.dev/pip-release-links.html \
+            --upgrade \
+            iree-compiler \
+            iree-runtime
+
+      - name: "Installing other Python requirements"
+        run: |
+          source ${VENV_DIR}/bin/activate
+          python3 -m pip install -r iree_tests/requirements.txt
       # TODO(scotttodd): add a local cache for these large files to a persistent runner
       - name: "Downloading remote files for real weight model tests"
         run: |
           source ${VENV_DIR}/bin/activate
-          python3 iree_tests/download_remote_files.py
+          python3 iree_tests/download_remote_files.py --root-dir pytorch/models
       - name: "Running real weight model tests"
+        env:
+            IREE_TEST_CONFIG_FILES: iree_tests/configs/config_sdxl_cpu_llvm_task.json
         run: |
           source ${VENV_DIR}/bin/activate
-          pytest iree_tests -n auto -k real_weights -rpfE --timeout=600 --retries 2 --retry-delay 5 --durations=0
+          pytest iree_tests/pytorch/models -s -n auto -k real_weights -rpfE --timeout=1200 --retries 2 --retry-delay 5 --durations=0
```

Comment:
The config file should match the tests that are running.

Either
* Rename the config file to `config_pytorch_models_cpu_llvm_task.json`
* Change the test to only run SDXL tests, e.g. `pytest -k sdxl` (possibly `pytest -k sdxl -k real_weights` ... not sure about the syntax for multiple filters)